docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,81 @@
+# Tutorial: Deploy the Penpot charm
+
+## What you'll do
+This tutorial will walk you through deploying the penpot charm; you will:
+1. Deploy and configure the penpot charm's dependencies (PostgreSQL, Redis, s3, etc.)
+2. Deploy Penpot and integrate it with its dependencies
+3. Create a user
+4. Login to the web interface for your deployed Penpot instance
+
+## Prerequisites
+* A kubernetes cluster.
+* A host machine with juju version 3.4 or above.
+
+## Create a juju model
+1. Create a juju model
+```
+juju add-model penpot-test
+```
+
+## Deploy and configure the penpot charm's dependencies
+1. Deploy and configure some dependencies
+```
+juju deploy minio --config access-key=minioadmin --config secret-key=minioadmin
+juju deploy postgresql-k8s --channel 14/stable --trust
+juju deploy redis-k8s --channel latest/edge
+juju deploy s3-integrator --config "endpoint=http://minio-endpoints.penpot-test.svc.cluster.local:9000" --config bucket=penpot
+juju deploy nginx-ingress-integrator --trust --config service-hostname=penpot.local --config path-routes=/
+juju deploy self-signed-certificates
+```
+
```

Comment:
Have a check here for `juju status` so that the user knows how to find out when the unit is active

Reply:
Sounds good, will do.